### DISCUSSION TITLE
CMS integration

### DISCUSSION THREAD

**Discussion by neo-art** [mearashadowfax/ScrewFast#61] [General]
Great theme, well done.

Just wondering if there is an easy way to add a CMS to the theme (Storyblok, Sanity etc.) to manage the blog posts and pages?

**Comment by mearashadowfax** [comment]
Thank you for your kind words about the theme, @neo-art ! While I haven't specifically tested integrating a CMS like Storyblok or others with the theme yet, I've taken a look at the [Astro Docs](https://docs.astro.build/en/guides/cms/) and it does seem quite manageable. Additionally, I noticed they offer Astro CMS integrations, including Storyblok-astro. You might find the process straightforward by following their documentation [here](https://github.com/storyblok/storyblok-astro).

**Reply by neo-art** [reply]
Thank you for pointing me in the right direction :) I tried to add Storyblok but as I am not a developer it took a lot of time and the result wasn't great :) Will continue working on it and thank you for the theme.
